constraint/pkg/apis/templates/v1/constrainttemplate_types_test.go

```diff
@@ -155,22 +181,22 @@ func TestValidationVersionConversionAndTransformation(t *testing.T) {
 		{
 			name: "Two deep properties, LegacySchema=true",
 			v: &Validation{
-				LegacySchema:    pointer.Bool(true),
```

Comment:
I feel like `pointer.Bool(true)` is a better tool, but the difference is minimal.

Reply:
This was a merge mistake